includes/class-wc-structured-data.php

```diff
@@ -275,38 +274,46 @@ public function generate_product_data( $product = null ) {
 				'reviewCount' => $product->get_review_count(),
 			);
 
-			// Markup most recent rating/review.
+			// Markup 5 most recent rating/review.
 			$comments = get_comments(
 				array(
-					'number'      => 1,
+					'number'      => 5,
 					'post_id'     => $product->get_id(),
 					'status'      => 'approve',
 					'post_status' => 'publish',
 					'post_type'   => 'product',
 					'parent'      => 0,
-					'meta_key'    => 'rating',
+					'meta_key'    => 'rating', // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key
```

Comment:
I didn't check, but I believe this is not necessary anymore now that you are using `meta_query`? If this is removed, you might need to change the value passed to `orderby` as well to reference the key in `meta_query`.

Reply:
I'll check again, maybe WPCS outdated? But I got a warning for it, that's why I included that notation.

Reply:
Ohh, you are talking about the `meta_key`? Since we are using `orderby` as `meta_value_num` we should set `meta_key` as well, at least is what WordPress documentation says.

Reply:
Also having both should at least get merged anyway, but this one is just for sorting.